index.html

```diff
@@ -1,199 +1,276 @@
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestiate to contact me.
+        </p>
+        <button type="submit" class="about-me-btn">Get My Resume</button>
       </div>
-      <div class="framework-card about-box">
-        <img src="images/Icon2.png" alt="icon for skills">
-        <h4 class="lang-title">Frameworks</h4>
-        <ul class="about-me-tags">
-          <li class="tag-about-items">Bootstrap</li>
-          <li class="tag-about-items">Ruby on rails</li>
-          <li class="tag-about-items">RSpec</li>
-          <li class="tag-about-items">Capybara</li>
-          <li class="tag-about-items">Salenium</li>
-        </ul>
+      <div class="grid-container">
+        <div class="lang-card about-box">
+          <img src="images/Icons.png" alt="icon languages" />
+          <h4 class="lang-title">Languages</h4>
+          <ul class="about-me-tags">
+            <li class="tag-about-items">JavaScript</li>
+            <li class="tag-about-items">Ruby</li>
+            <li class="tag-about-items">HTML</li>
+            <li class="tag-about-items">CSS</li>
+          </ul>
+        </div>
+        <div class="framework-card about-box">
+          <img src="images/Icon2.png" alt="icon for skills" />
+          <h4 class="lang-title">Frameworks</h4>
+          <ul class="about-me-tags">
+            <li class="tag-about-items">Bootstrap</li>
+            <li class="tag-about-items">Ruby on rails</li>
+            <li class="tag-about-items">RSpec</li>
+            <li class="tag-about-items">Capybara</li>
+            <li class="tag-about-items">Salenium</li>
+          </ul>
+        </div>
+        <div class="skills-card about-box">
+          <img src="images/icon-skills.png" alt="icon for skills" />
+          <h4 class="lang-title">Skils</h4>
+          <ul class="about-me-tags">
+            <li class="tag-about-items">codeKit</li>
+            <li class="tag-about-items">GitHub</li>
+            <li class="tag-about-items">codepen</li>
+            <li class="tag-about-items">Gitlap</li>
+            <li class="tag-about-items">Terminal</li>
+          </ul>
+        </div>
       </div>
-      <div class="skills-card about-box">
-        <img src="images/icon-skills.png" alt="icon for skills">
-        <h4 class="lang-title">Skils</h4>
-        <ul class="about-me-tags">
-          <li class="tag-about-items">codeKit</li>
-          <li class="tag-about-items">GitHub</li>
-          <li class="tag-about-items">codepen</li>
-          <li class="tag-about-items">Gitlap</li>
-          <li class="tag-about-items">Terminal</li>
+    </section>
+
+    <section class="form-section">
+      <div class="form-container">
+        <h2 class="form-title">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form action="https://formspree.io" method="post">
```

Comment:
- [ ] Kindly be noticed that you should provide a `form_id` to the `action` URL ie.
```html
<form action="https://formspree.io/f/{form_id}" method="post">
```